xcode/Subconscious/Shared/Components/Detail/UserProfileDetailMetaSheet.swift

```diff
@@ -0,0 +1,298 @@
+//
+//  UserProfileMetaSheet.swift
+//  Subconscious (iOS)
+//
+//  Created by Ben Follington on 29/3/2023.
+//
+
+import SwiftUI
+import ObservableStore
+
+struct MetaTableItemShareLinkView: View {
+    var label: String
+    var item: String
+    
+    var body: some View {
+        Button(
+            action: {}, // Action is handled by ShareLink
+            label: {
+                ShareLink(item: item) {
```

Comment:
`.buttonStyle` works on `ShareLink`. So we can get rid of the wrapping button and use ShareLink directly.

```swift
ShareLink(item: item) {
    Label(
        label,
        systemImage: "square.and.arrow.up"
    )
}
.buttonStyle(RowButtonStyle())
```

Reply:
I _swear_ I tried this before anything else and couldn't get it to behave it must've been a dream, it works fine now 🤦